package.json

```diff
@@ -62,6 +62,7 @@
     "karma-sourcemap-loader": "^0.3.6",
     "karma-spec-reporter": "0.0.23",
     "karma-webpack": "^1.7.0",
+    "lodash": "^4.11.1",
```

Comment:
Does webpack do tree-shaking? If not, perhaps we want to just include the subpackages for the particular functions we care about to reduce vendor dependency size?